src/SFA.DAS.AssessorService.Data/SettingRepository.cs

```diff
@@ -44,15 +44,17 @@ public async Task SetSetting(string name, string value)
                     $@"VALUES (@value, @name)",
                     param: new { value, name },
                     transaction: _unitOfWork.Transaction);
+                
+                return true;
             }
-            else
-            {
-                await _unitOfWork.Connection.ExecuteAsync(
+         
+            await _unitOfWork.Connection.ExecuteAsync(
                 "UPDATE [Settings] SET [Value] = @value " +
                 "WHERE [Name] = @name",
                 param: new { value, name },
                 transaction: _unitOfWork.Transaction);
-            }
+            
+            return false;
```

Comment:
the caller of this class would have no idea what a true result means without drilling down and inspecting the code.

Reply:
Refactored to externalise the result